### PR TITLE
[X pipeline]Update public documentation

### PR DESCRIPTION
**Pull request by alex-vazquez**
---
### Purpose of this PR
Fix missing documentation for some API changes that were recently introduced.

Added a checkbox on the DocChecker to allow outputing the info on the output

image

JIRA: https://jira.unity3d.com/browse/XPIPELINE-146

---
### Testing status
Checking the HDRP_Tests project -> Window -> DocChecker and see that my documentation warnings are not longer visible

---

### CODE REVIEW

File: com.unity.render-pipelines.core/Editor/Camera/ISerializedCamera.cs [hunk 1]
```diff
@@ -1,24 +1,42 @@
...
+        /// <summary>Allow Dynamic resolution property</summary>
         SerializedProperty allowDynamicResolution { get; }
+        /// <summary>Volume layer mask property</summary>
         SerializedProperty volumeLayerMask { get; }
+        /// <summary>Clear Depth property property</summary>
         SerializedProperty clearDepth { get; }
+        /// <summary>Anti aliasing property</summary>
         SerializedProperty antialiasing { get; }
 
+        /// <summary>Method that updates the <see cref="SerializedObject"/> of the Camera and the Additional Camera</summary>
```
Comment (RSlysz) [inline com.unity.render-pipelines.core/Editor/Camera/ISerializedCamera.cs:33]: and the Additional Camera _**Data**_

File: com.unity.render-pipelines.core/Editor/Camera/ISerializedCamera.cs [hunk 2]
```diff
@@ -1,24 +1,42 @@
...
         void Update();
+
+        /// <summary>Applies the modified properties to the <see cref="SerializedObject"/> of the Camera and the Additional Camera</summary>
```
Comment (RSlysz) [inline com.unity.render-pipelines.core/Editor/Camera/ISerializedCamera.cs:36]: same

File: com.unity.render-pipelines.core/Editor/Camera/ISerializedCamera.cs [hunk 3]
```diff
@@ -1,24 +1,42 @@
...
         void Apply();
+
+        /// <summary>Refreshes the <see cref="SerializedProperty"/> of the <see cref="SerializedObject"/> of the Camera and the Additional Camera</summary>
```
Comment (RSlysz) [inline com.unity.render-pipelines.core/Editor/Camera/ISerializedCamera.cs:39]: same

File: com.unity.render-pipelines.core/Editor/CoreEditorStyles.cs [hunk 4]
```diff
@@ -27,9 +27,13 @@ public static class CoreEditorStyles
         /// <summary><see cref="Texture2D"/> 1x1 pixel with blue color</summary>
         public static readonly Texture2D blueTexture;
 
-        /// <summary> PaneOption icon </summary>
+        /// <summary> PaneOption icon for dark skin</summary>
         static readonly Texture2D paneOptionsIconDark;
+
+        /// <summary> PaneOption icon for light skin</summary>
```
Comment (RSlysz) [inline com.unity.render-pipelines.core/Editor/CoreEditorStyles.cs:33]: Not required as not public :) (can keep it)

**Review by RSlysz — APPROVED** [review]
Add missing "Data" (x3) and it will be good

File: com.unity.render-pipelines.core/Editor/CoreEditorStyles.cs (hunk 4 above)
Comment (alex-vazquez) [reply com.unity.render-pipelines.core/Editor/CoreEditorStyles.cs:33]: I was in the mood to document everything xD

**Review by JulienIgnace-Unity — APPROVED** [review]

**Comment by alex-vazquez** [comment]
@Unity-Technologies/gfx-qa-hdrp , can you go to HDRP_Tests > Window > Doc Checker and see if the checkbox that I've added is showing the undocumented clases on the console log?

Thank you

**Review by RemyUnity — APPROVED** [review]
LGTM.
On a side note, is it really necessary to have the doc checker in the HDRP graphic tests project ? I mean, do we want to pay the time cost of opening and importing the graphic tests for checking doc ?

**Pull request merged**